README.md

```diff
@@ -41,67 +32,6 @@ Requires that you [install Replayable Desktop](https://replayable.io). Replayabl
 
 Also see [the examples folder](https://github.com/replayableio/cli/tree/main/examples).
 
-## Web
-
-### Setup
-
-Nothing! The app exposes the protocol to the system natively via `replayable://`.
-
-### HTML Anchor Tag
-
-```html
-<a href="replayable://replay/create" target="_blank">Create a Replay</a>
-```
-
-### JS Error Handler
-
-```js
-window.onerror = function myErrorHandler() {
-  window.open("replayable://replay/create", "_blank");
-};
-
-setTimeout(() => {
-  throw new Error("Throw makes it go boom!");
-}, 3000);
-```
-
-## NodeJS SDK
-
-### Setup
-
-```sh
-npm install replayable
-```
-
-### Create a Replay
-
-```js
-const replayable = require("replayable");
-
-let replay = await replayable.createReplay({
-  title: "My New Replay",
-  description: `This **renders markdown** or plaintext in monospace font.`
-});
-```
-
-### Error Handler
-
-```js
-const replayable = require("replayable");
-
-process.on("uncaughtException", async (err) => {
-  let replay = await replayable.createReplay({
-    title: "uncaughtException",
-    description: err,
-  });
-  console.log("Replayable", replay);
-});
-
-setTimeout(() => {
-  throw new Error("Throw makes it go boom!");
-}, 3000);
-```
-
```

Comment:
All NodeJS is within the scope of this package. NodeJS packages can be a CLI, frontend, or backend.